### PR TITLE
Snake border issue

### PR DESCRIPTION
**Pull request by hvirmani**
Issue:

### CODE REVIEW

**Comment by hvirmani** [comment]
Snake border button issue has been resolved.

**Review by shahednasser — CHANGES_REQUESTED** [review]
I like the idea of the button but I have a few remarks:

1. Change the name of the class to `snake-btn` to match the naming convention of classes.
2. I'm confused about how you compiled the snake-btn considering you didn't import it in `src/components/_animated.less` Did you edit `dist/sbuttons.css` yourself? Regardless of how, Please import `_snake.less` in `_animated.less` and then run `npm run compile` to generate `dist/sbuttons.css`.
3. Instead of including rules relating to `[data-theme="dark"]`, I think it's best for the snake border color to be based on the color of the button itself. If the color is light, use dark border and vice versa. The reason behind this is that `[data-theme="dark"]` is only used on our website, but if someone were to use sButtons in their projects it would not apply for them.
4. If you added this button idea in the button ideas file make sure to check it off there.

If you need any assistance or have any questions please let me know.

**Comment by hvirmani** [comment]
I have done suggested changes.

File: src/components/animated/_snake.less [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    }
+    &:after {
+      width: 100%;
+      height: 100%;
+      opacity: 1;
+      border-bottom: 0.2rem solid;
+      border-left: 0.2rem solid;
+      transition: width 300ms cubic-bezier(0.07, 0.62, 0.61, 1),
+        height 150ms 300ms cubic-bezier(0.07, 0.62, 0.61, 1);
+      border-color: #9a9;
```
Comment (shahednasser) [inline src/components/animated/_snake.less:47]: Please change border color to `#333`

File: src/components/animated/_snake.less [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  &:hover {
+    &:before {
+      width: 100%;
+      height: 100%;
+      opacity: 1;
+      border-top: 0.2rem solid yellow;
+      border-right: 0.2rem solid yellow;
+      transition: width 300ms cubic-bezier(0.07, 0.62, 0.61, 1),
+        height 150ms 300ms cubic-bezier(0.07, 0.62, 0.61, 1);
+      border-color: #9a9;
```
Comment (shahednasser) [inline src/components/animated/_snake.less:37]: Please change border color to `#333`

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: src/components/animated/_snake.less (hunk 2 above)
Comment (hvirmani) [reply src/components/animated/_snake.less:37]: okay

**Comment by hvirmani** [comment]
Done

**Comment by shahednasser** [comment]
Awesome thank you so much!

@all-contributors please add @hvirmani for code

**Pull request merged**